include/aspect/particle/particle_handler.h

```diff
@@ -133,6 +140,15 @@ namespace aspect
         particle_iterator_range
         particle_range_in_cell(const typename parallel::distributed::Triangulation<dim,spacedim>::active_cell_iterator &cell);
 
+
+        /**
+         * Return a pair of particle iterators that mark the begin and end of
+         * the particles in a particular cell. The last iterator is the first
+         * particle that is no longer in the cell.
+         */
+        particle_iterator_range
+        particle_range_in_cell(const typename parallel::distributed::Triangulation<dim,spacedim>::active_cell_iterator &cell) const;
```

Comment:
That name caught me off guard a bit because I thought that the "range is in the cell". How about `particles_in_cell`? Or `particles_in_cell_range`?